clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java

```diff
@@ -11,6 +11,7 @@ public enum Arg {
   URL("url", "u", true, "Venice url, eg. http://localhost:1689  This can be a router or a controller"),
   VENICE_ZOOKEEPER_URL("venice-zookeeper-url", "vzu", true, "Venice Zookeeper url, eg. localhost:2622"),
   CLUSTER("cluster", "c", true, "Name of Venice cluster"),
+  TOKEN("token", "t", true, "Authentication Token for JWT authentication"),
```

Comment:
IMHO for users who use always a "token" it is more user-friendly "--token" (json-web- is redundant).

Reply:
in the community meeting we decided to not expose this parameter but read the credentials from the ssl config file https://github.com/linkedin/venice/blob/4738167344a7884655fea6263ea8d745a267eeec/clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java#L149C20-L149C35 

in the future we can rename the option to "auth-file" or something like that

Reply:
@FelixGV I have changed the code in order to read the Authentication configuration from the SSL config file.

it actually looks better than before, because this way it will be easier to pass any configurations to the ClientAuthenticationProviderFactory and we are not sticking the CLI tool to a specific AuthenticationProvider